src/renderer/gl_engine/tvgGlShaderSrc.cpp

```diff
@@ -509,3 +509,90 @@ const char* BLIT_FRAG_SHADER = TVG_COMPOSE_SHADER(
         FragColor = texture(uSrcTexture, vUV);                      \n
     }
 );
+
+const char* COMPLEX_BLEND_FRAG_SHADER = TVG_COMPOSE_SHADER(
```

Comment:
Is it possible too separate blend methods by the different fragment shaders insted of cheking for each pixel?